css-tailwind-intro/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css"
+      rel="stylesheet" />
+    <title>Tailwind CSS Website Layout</title>
+  </head>
+  <body class="bg-gray-100">
+    <div class="container mx auto p-4">
+      <div class="bg-blue-500 text-white p-4 rounded-lg shadow-lg">
+        <h1 class="text-3xl font-bold">I am the header!</h1>
+      </div>
+      <div class="flex mt-4">
+        <div class="w-1/4 bg-gray-200 p-4 rounded-lg shadow-lg">
+          <h2 class="text-xl font-bold">I am a sidebar!</h2>
+          <ul>
+            <li class="mt-2">
+              <a href="" class="text-blue-500">Link 1</a>
+            </li>
+            <li class="mt-2">
+              <a href="" class="text-blue-500">Link 2</a>
+            </li>
+            <li class="mt-2">
+              <a href="" class="text-blue-500">Link 3</a>
+            </li>
+          </ul>
+        </div>
+        <div class="w-3/4 bg-white p-4 ml-4 rounded-lg shadow lg">
+          <h2 class="text-cl font-bold">I am the main content!</h2>
```

Comment:
This is likely a type and should be `text-xl`

Reply:
Whoops! I'll fix that.